### PR TITLE
:Bug:Fix signin-widget label customization for locale codes like 'xx-XX'

### PR DESCRIPTION
**Pull request by eugenebakaiev-okta**
Added adjustLocalizationKey method which converts language code to a style which is used in overrides array (eg 'nl-nl', 'nl-NL')

Resolves: OKTA-138149

### CODE REVIEW

**Comment by eugenebakaiev-okta** [comment]
@hor-kanchan-okta, @upteam-okta please review this PR

File: src/util/Bundles.js [hunk 1]
```diff
@@ -196,18 +196,49 @@ define([
...
+            return keyPart;
+          }).join('-');
+        }
+      });
+
+      return adjustedLocalizationKey;
+    },
+
     loadLanguage: function (language, overrides, assets) {
       var parsedOverrides = parseOverrides(overrides);
```
Comment (hor-kanchan-okta) [inline src/util/Bundles.js]: The other solution is to modify `parseOverrides`. Always return the key in lowercase. The comparison will be case insensitive. Will let @samerfanek-okta to decide

Comment (samerfanek-okta) [reply src/util/Bundles.js]: Yeah I think it makes more sense to put it in ``parseOverrides`` and return a lowercase key. Also, can you add unit tests for this covering the various combinations? (For example, 'nl', 'NL', 'nl-nl', 'nl-NL', etc..)

**Comment by eugenebakaiev-okta** [comment]
@hor-kanchan-okta @samerfanek-okta updated PR please look

**Pull request merged**